packages/react/src/tabs/root/TabsRoot.test.tsx

```diff
@@ -1221,4 +1221,189 @@ describe('<Tabs.Root />', () => {
       expect(secondTab).to.have.attribute('aria-selected', 'false');
     });
   });
+
+  describe('disabled tab handling', () => {
+    it('should select the first non-disabled tab by default when defaultValue is not provided', async () => {
+      const { getAllByRole } = await render(
+        <Tabs.Root>
+          <Tabs.List>
+            <Tabs.Tab value={0} disabled data-testid="tab-0">
+              Tab 0
+            </Tabs.Tab>
+            <Tabs.Tab value={1} data-testid="tab-1">
+              Tab 1
+            </Tabs.Tab>
+            <Tabs.Tab value={2} disabled data-testid="tab-2">
+              Tab 2
+            </Tabs.Tab>
+            <Tabs.Tab value={3} data-testid="tab-3">
+              Tab 3
+            </Tabs.Tab>
+          </Tabs.List>
+          <Tabs.Panel value={0}>Panel 0</Tabs.Panel>
+          <Tabs.Panel value={1}>Panel 1</Tabs.Panel>
+          <Tabs.Panel value={2}>Panel 2</Tabs.Panel>
+          <Tabs.Panel value={3}>Panel 3</Tabs.Panel>
+        </Tabs.Root>,
+      );
+
+      const tabs = getAllByRole('tab');
+
+      // The first non-disabled tab (tab 1) should be selected
+      expect(tabs[1]).to.have.attribute('aria-selected', 'true');
+      expect(tabs[0]).to.have.attribute('aria-selected', 'false');
+      expect(tabs[2]).to.have.attribute('aria-selected', 'false');
+      expect(tabs[3]).to.have.attribute('aria-selected', 'false');
+    });
+
+    it('should select the first non-disabled tab when first two tabs are disabled', async () => {
```

Comment:
WDYT about aligning the wording on both the expect and condition parts? 🤔 
```suggestion
    it('should select the third tab when first two tabs are disabled', async () => {
```